google/cloud/bigtable/client_options.h

```diff
@@ -83,6 +83,15 @@ class ClientOptions {
     return *this;
   }
 
+  /// Return the current endpoint for instance admin RPCs.
+  std::string const& instance_admin_endpoint() const {
```

Comment:
If these two methods are purely to facilitate internal testing and do not add any user-visible value otherwise (i.e., we don't expect users to use it), would it make sense to make this `protected` and have a friend class use these methods, or am I overthinking it?

Just thinking that maybe (maybe) at some point (in the far distant future), if we fix the emulator, we would probably want to delete these methods, which would be easier if they're `protected` and much harder if they're `public`.

Or maybe that's just too much work? Am I overthinking this?